packages/caliper-core/lib/common/config/Config.js

```diff
@@ -54,8 +54,12 @@ const keys = {
         }
     },
     Monitor: {
-        DefaultInterval: 'caliper-monitor-default-interval',
-        PrometheusScrapePort: 'caliper-monitor-prometheus-scrape-port'
+        Default: {
+            Interval: 'caliper-monitor-default-interval'
```

Comment:
Seems a bit odd to have a default section if the whole thing is to provide default values